src/locales/locale-en.json

```diff
@@ -106,7 +106,7 @@
     "UPCOMING_ACTIVITIES_IN_SERIES": "Upcoming activities in this series"
   },
   "ACTIVITY_FEEDBACK": {
-    "AMOUNT": "Amount of food you were able to save (in kg)",
+    "AMOUNT": "Amount of food you were able to save",
```

Comment:
Given we are changing the text, thus trigger the translations to be redone, I wonder if it makes sense to make it non-food specific? What do you think @brnsolikyl?

Could be _Amount you were able to save_ ... well, even _save_ might not be the right term for some cases, now it could be more general... but one step at a time...